Added trial information to VirtualHere (#707)
MichaIng/DietPi-Docs: docs/software/remote_desktop.md

![VirtualHere client screenshot](../assets/images/dietpi-software-remotedesktop-virtualhere.png){: width="400" height="252" loading="lazy"}

The functionality consists of two parts:

> - The [server side](https://virtualhere.com/content/usb-servers): This software part is installed within this DietPi software package. It serves your USB device over the network

Reviewer: ```suggestion
- The [server](https://virtualhere.com/content/usb-servers): This software part is installed with the DietPi software package. It serves your USB device over the network
```

> - The [client side](https://virtualhere.com/usb_client_software): This software part needs to be installed on every client who wants to access the USB devices

Reviewer: ```suggestion
- The [client](https://virtualhere.com/usb_client_software): This software part needs to be installed on every client which wants to access the USB devices.
```
